<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>200</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionnew"/>
    <addaction name="actionopen"/>
    <addaction name="separator"/>
    <addaction name="actionsave"/>
    <addaction name="actionsaveAs"/>
    <addaction name="separator"/>
    <addaction name="actioncloseAll"/>
    <addaction name="actionquit"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actioncut"/>
    <addaction name="actioncopy"/>
    <addaction name="actionpaste"/>
    <addaction name="separator"/>
    <addaction name="actionundo"/>
   </widget>
   <widget class="QMenu" name="menu_show">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="actiontool"/>
    <addaction name="actionstatus"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_show"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionnew">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionsaveAs">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actioncloseAll">
   <property name="text">
    <string>全部关闭</string>
   </property>
  </action>
  <action name="actionquit">
   <property name="text">
    <string>退出程序</string>
   </property>
  </action>
  <action name="actioncut">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="actioncopy">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actionpaste">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actionundo">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="actiontool">
   <property name="text">
    <string>工具栏</string>
   </property>
  </action>
  <action name="actionstatus">
   <property name="text">
    <string>状态栏</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
